src/scripts/Common/Base.cpp

```diff
@@ -679,6 +679,53 @@ EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, const char* pTex
     return NewEmote;
 }
 
+EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, uint32 scripttext)
+{
+    EmoteDesc* NewEmote = NULL;
+    NpcScriptText const* ct = sMySQLStore.GetNpcScriptText(scripttext);
+    TextType pType;
+
+    switch (ct->type)
```

Comment:
nullptr dereference.
ct might be nullptr, you use it here before you know it is not nullptr (See line 704).